lib/main.rb

```diff
@@ -1,3 +1,19 @@
-class Fibonacci 
-    
-end 
\ No newline at end of file
+class Puzzle 
+
+    def fibonacci(num)
+        start_arr = [0,1]
+        while start_arr.size < num
+            sum = start_arr[start_arr.size - 1] + start_arr[start_arr.size - 2]
+            start_arr.push(sum)
+        end
+        
+        puts start_arr
+        return start_arr
+
+    end
+
+end 
+
+puzzle = Puzzle.new
```

Comment:
What are your thoughts on moving this to a separate file, or alternatively moving the class to it's own file?

Reply:
Yes, I think it's a good idea, I was thinking about extracting everything out into their own files.